### PR TITLE
Expose a JSON api to query for benchmark results

### PR DESCRIPTION
**Pull request by jerryliu55**
Normally, the web app uses a url like 
`https://rubybench.org/ruby/ruby/releases?result_type=app_fib` 
to display benchmark results

This PR allows us to get JSON versions of the results by sending requests to
`https://rubybench.org/ruby/ruby/releases.json?result_type=app_fib`

An example of what the JSON results looks like:
https://gist.github.com/jerryliu55/e89e007386018977b6b8fa87c9e71f0f

### CODE REVIEW

**Comment by tgxworld** [comment]
Can we add some simple tests for this? Thank you @jerryliu55

**Comment by jerryliu55** [comment]
@tgxworld Added some tests

**Comment by tgxworld** [comment]
Hi @jerryliu55, can you base so that the tests are passing. Thank you for all your work on RubyBench so far :+1:

File: app/controllers/concerns/json_generator.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        benchmark_name: params[:result_type],
+        datapoints: datapoints,
+        measurement: result_type[:name],
+        unit: result_type[:unit]
+      }
+      config[:variations] = variations if variations.length > 1
+      config
+    end
+  end
+end
```
Comment (tgxworld) [inline app/controllers/concerns/json_generator.rb]: Nitpick: Missing blank line at the end here

**Comment by jerryliu55** [comment]
@tgxworld closing this since I did a rewrite which I think is a much better solution: https://github.com/ruby-bench/ruby-bench-web/pull/192

**Pull request closed without merging**